scripts/initialize_universal_profiling.sh

```diff
@@ -0,0 +1,11 @@
+set -eo pipefail
+if [ -z "$ELASTIC_PASSWORD" ]; then
+    echo "Set the ELASTIC_PASSWORD environment variable in the .env file" >&2;
+    exit 1;
+fi;
+curl --no-progress-meter --fail -X POST \
+    --cacert /usr/share/elasticsearch/config/certs/ca/ca.crt \
+    --user "elastic:$ELASTIC_PASSWORD" \
+    -H "Content-Type: application/json" \
+    -H "kbn-xsrf: true" \
+    https://kibana:$KIBANA_PORT/internal/profiling/setup/es_resources
```

Comment:
Is this the only way to do this or is there some ES or kibana yml config that could make it so we don't have to make an API call?

Reply:
this is the only way to set up universal profiling... it has to be manually triggered.